frappe/core/api/file.py

```diff
@@ -17,7 +17,7 @@ def unzip_file(name: str):
 def get_attached_images(doctype: str, names: list[str] | str) -> frappe._dict:
 	"""get list of image urls attached in form
 	returns {name: ['image.jpg', 'image.png']}"""
-
+	file_type = ["PNG", "JPEG", "JPG", "GIF", "BMP", "WEBP"]
```

Comment:
This list feels a little restrictive. Can we use a broader list, like `Pillow`'s (`from PIL.Image import EXTENSION, init`) or `mimetypes`  (`_default_mime_types`)?

Reply:
In that case, this should do 
```python
exts = Image.registered_extensions()
supported_extensions = {f for ex, f in exts.items() if f in Image.OPEN}
```